scripts/measures/initialize-measures-data.ts

```diff
@@ -117,6 +118,25 @@ function initializeSevenPointCapRemoved() {
     }
 }
 
+function removeStrataForSPR() {
+    const year = parseInt(performanceYear, 10);
+
+    if (isNaN(year) || year <= 2025) {
```

Comment:
I think you can remove this check, since the init script will never be run for years prior to 2026.